quiz.rb

```diff
@@ -14,24 +14,30 @@
 # Take a string and return an array of normalized words.
 # normalized words contain no punctuation and are all lower case
 def normalize_words words
+  paragraph.split/\W+/
```

Comment:
the first line of this method will do the trick, but the second like is superfluous and breaks your code